exercises/yacht/introduction.md

```diff
@@ -0,0 +1,7 @@
+# Introduction
+
+The dice game [Yacht][yacht] is from the same family as Poker Dice, Generala and particularly Yahtzee, of which it is a precursor.
+In the game, five dice are rolled and the result can be entered in any of twelve categories.
+The score of a throw of the dice depends on the category chosen.
```

Comment:
```suggestion
```suggestion
The game consists of twelve rounds.
In each, five dice are rolled and the player chooses one of twelve categories.
The chosen category is then used to score the throw of the dice.
```


```